app/models/ability.rb

```diff
@@ -155,6 +158,8 @@ def signed_in_with_cfp_role(user)
     can :manage, CallForPaper, conference_id: conf_ids_for_cfp
     can :manage, Commercial, commercialable_type: 'Event',
                              commercialable_id: Event.where(conference_id: conf_ids_for_cfp).pluck(:id)
+    can :index, Comment, commentable_type: 'Event',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.